next/frontend/utils/form.ts

```diff
@@ -71,12 +57,27 @@ export const getAjvFormKeywords = (
     },
   ]
 }
-export const ajvFormats = {
-  // TODO: Explore if only Slovak zip codes are supported
+
+// Copy of https://github.com/ajv-validator/ajv-formats/blob/4dd65447575b35d0187c6b125383366969e6267e/src/formats.ts#L180C1-L186C2
+function compareTime(s1: string, s2: string): number | undefined {
+  if (!(s1 && s2)) return undefined
+  const t1 = new Date(`2020-01-01T${s1}`).valueOf()
+  const t2 = new Date(`2020-01-01T${s2}`).valueOf()
+  if (!(t1 && t2)) return undefined
+  return t1 - t2
+}
+
+export const ajvFormats: CurrentOptions['formats'] = {
   zip: /\b\d{5}\b/,
   // TODO: Remove, but this is needed for form to compile
   ciselnik: () => true,
-  localTime: () => true,
+  // https://blog.kevinchisholm.com/javascript/javascript-e164-phone-number-validation/
+  'phone-number': /^\+?[1-9]\d{1,14}$/,
```

Comment:
if we're doing it this way (=regex way), then let's remove the libphonenumber validation elsewhere which passes a wider range of numbers (I agree that we don't need to allow those). May need a very slight update to error translations ('number without spaces').

Reply:
Created issue
https://github.com/bratislava/konto.bratislava.sk/issues/330